prospector/commit_processor/feature_extractor.py

```diff
@@ -42,3 +44,7 @@ def extract_changes_relevant_path(
     of relevant paths (mentioned in the advisory record)
     """
     return any([changed_path in relevant_paths for changed_path in changed_paths])
+
+
+def extract_contains_jira_reference(jira_references: "list[str]") -> bool:
```

Comment:
Isn't this supposed to take a commit message in input? that is, a str, rather than a list[str]?

Reply:
The preprocessor extracts the list of JIRA references from the commit message. Here we already have the pre-processed commit with the extracted list of JIRA references, that's why we don't need to extract them from string here.